openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryV1.component.tsx

```diff
@@ -327,32 +401,41 @@ const GlossaryV1 = ({
           />
         </div>
         <div className="tw-relative tw-mr-2 tw--mt-2" id="add-term-button">
-          <NonAdminAction position="bottom" title={TITLE_FOR_NON_ADMIN_ACTION}>
-            <Button
-              className={classNames('tw-h-8 tw-rounded tw-mb-1 tw-mr-2', {
-                'tw-opacity-40': isHasAccess,
-              })}
+          <Tooltip
+            title={
+              createGlossaryTermPermission
+                ? 'Add Term'
+                : NO_PERMISSION_FOR_ACTION
+            }>
+            <ButtonAntd
+              className="tw-h-8 tw-rounded tw-mb-1 tw-mr-2"
               data-testid="add-new-tag-button"
-              size="small"
-              theme="primary"
-              variant="contained"
+              disabled={!createGlossaryTermPermission}
+              type="primary"
               onClick={handleAddGlossaryTermClick}>
               Add term
-            </Button>
-          </NonAdminAction>
-          <NonAdminAction position="bottom" title={TITLE_FOR_NON_ADMIN_ACTION}>
-            <Dropdown
-              align={{ targetOffset: [-12, 0] }}
-              overlay={manageButtonContent()}
-              overlayStyle={{ width: '350px' }}
-              placement="bottomRight"
-              trigger={['click']}
-              visible={showActions}
-              onVisibleChange={setShowActions}>
+            </ButtonAntd>
+          </Tooltip>
+
+          <Dropdown
+            align={{ targetOffset: [-12, 0] }}
+            disabled={!glossaryPermission.Delete}
+            overlay={manageButtonContent()}
+            overlayStyle={{ width: '350px' }}
+            placement="bottomRight"
+            trigger={['click']}
+            visible={showActions}
+            onVisibleChange={setShowActions}>
+            <Tooltip
+              title={
+                glossaryPermission.Delete
+                  ? 'Manage Glossary'
```

Comment:
Is this text correct?

Reply:
Yes, There we have a button with three vertical dots.
<img width="434" alt="image" src="https://user-images.githubusercontent.com/59080942/187517505-1bd89720-b890-4c91-a99a-b439e5c0973f.png">

Reply:
So we should't disable it based on Delete permission..
We can disable delete option in it but not the more option based on delete permission

Reply:
Okay, got it!.